src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerableFactory.cs

```diff
@@ -34,17 +40,28 @@ internal static void NormalizeInputs(ref string directory, ref string expression
 
             ReadOnlySpan<char> directoryName = Path.GetDirectoryName(expression.AsSpan());
 
+            bool isNormalized = true;
```

Comment:
`isNormalized` doesn't make sense anymore- you weren't passed any context. 'directoryModified' perhaps?

Reply:
Makes sense. Will rename it.